src/ruby.rs

```diff
@@ -44,7 +44,148 @@ impl_from_arity!(from_arity_13, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE,
 impl_from_arity!(from_arity_14, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE);
 impl_from_arity!(from_arity_15, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE, VALUE);
 
+#[repr(transparent)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[allow(non_camel_case_types)]
+pub struct ruby_value_type(c_int);
+
 extern {
+    #[link_name = "RS_T_NONE"]
+    pub static RB_T_NONE: ruby_value_type;
```

Comment:
Can we generate definitions link for these things?

Reply:
Also, I think you meant `RUBY_T_*` here. I suspect Ruby had both aliases for historical/compatibility reasons. Since we don't have that baggage, we should just investigate which one is the "more public (recent?)" API and pick one.

Reply:
What sort of docs would be useful here?

Reply:
Also, should we just drop the prefixed versions?